.vscode/settings.json

```diff
@@ -3,6 +3,7 @@
   "deno.lint": true,
   "deno.importMap": "./.vscode/import_map.json",
   "deno.codeLens.test": true,
+  "deno.documentPreloadLimit": 2000,
```

Comment:
Looks like this PR crossed the default `1000` file limit. Needed to up this setting, otherwise intellisense stops working.